js/PulsarFormComponent.js

```diff
@@ -1,10 +1,11 @@
 'use strict';
 
 var $ = require('jquery'),
-    TimePickerComponent = require('./TimePickerComponent');
+TimePickerComponent = require('./TimePickerComponent');
```

Comment:
Not important but we normally tab these var declarations out like prior to your change.